src/components/DateRangeInput/DateRangeInput.test.tsx

```diff
@@ -248,6 +248,72 @@ it.only('allows changing time options in 24h mode', async () => {
   expect(ending.format('DD/MM/YYYY HH:mm A')).toBe('11/11/2020 00:22 AM');
 });
 
+it.only('rounds start and end dates by minute', async () => {
```

Comment:
You forgot an `.only` here :D

Reply:
I added `.only` because most of the tests in that file are skipped.